lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
```

Comment:
Here's my take on what was discussed

Immediate actions: 
* add a `scorePrecision` or similarly named property to the performance audits `meta` object specifying a `numeric` precision
* add default `scorePrecision` of `binary` to the result
* update ReportRenderer to use the `audit.result.scorePrecision` combined with `audit.score` to decide how to display (no more `typeof score === 'boolean'` outside of ReportGenerator)

Potential future course of action that need further discussion: We remove `score` from the audit result object and instead have them return values that are compared to the declared target/point of diminishing returns/median values and a score is generated for them, boolean audits return whether they pass or fail as they do today and a score of 100/0 is generated for them. We explore the possibility of introducing letter grades and changing the score curves.